src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -40,46 +46,52 @@ protected override string BuildChallengeUrl(AuthenticationProperties properties,
             var address = QueryHelpers.AddQueryString(Options.AuthorizationEndpoint, new Dictionary<string, string>
             {
                 ["app_id"] = Options.ClientId,
-                ["scope"] = FormatScope(),
+                ["scope"] = string.Join(",", Options.Scope),
```

Comment:
What was the reason for removing the `FormatScope()` override?

Reply:
Only 1 time reference, keep override?

Reply:
As the provider base class already provides functionality to do this, it's better to override it.

Reply:
OK. Thanks. :)